tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, addr, log_requests=1):
         # to True to allow quick restart on the same port.  This is equivalent 
         # to calling setsockopt(SOL_SOCKET,SO_REUSEADDR,1)
         self.allow_reuse_address = True
+        # Increase request_queue_size to handle issues with many simultaneous
+        # connections in OSX 10.11
+        self.request_queue_size = 128
```

Comment:
Instead of hard coding a value (which might not be supported) this could set the queue size to e.g. `min(socket.SOMAXCONN, 128)`.